BDD/src/step-definitions/steps.js

```diff
@@ -0,0 +1,88 @@
+import { Given, When, Then, And } from '@wdio/cucumber-framework';
+import { expect, $, browser } from '@wdio/globals';
+
+import { MainPage } from '../pages/mainPage';
+import { DashboardPage } from '../pages/dashboardPage';
+import { LoginHelper } from '../utils/helpers/loginHelper';
+
+const mainPage = new MainPage();
+const dashboardPage = new DashboardPage();
+const loginHelper = new LoginHelper();
+
+Given('I open web site {string}', async page => {
```

Comment:
page isn't a meaningful name in this context. It's better to use something more descriptive, like pageUrl